Bottom: 833e47379fe7fcc61b67799010094ff5c5c47a58
Top:    b7057d4d49eb9868fa5cf0a5d540c5887c2c82cf
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 01:34:23 +0000

Fix make distcheck


---

diff --git a/w32/Makefile.am b/w32/Makefile.am
index af86e32..5355f61 100644
--- a/w32/Makefile.am
+++ b/w32/Makefile.am
@@ -2,8 +2,6 @@ EXTRA_DIST= build-all.sh minipack.conf mpk README \
   patches/gd patches/gd/0001-Export-sysbols-when-building-as-a-DLL.patch \
   patches/gd/0002-Libtool-requires-no-undefined-to-build-a-DLL.patch \
   patches/gd/0003-Avoid-conflit-between-libjpeg-and-windows-headers.patch \
-  patches/gettext/01-revert-sed-string.patch \
-  patches/gettext/02-Avoid-missing-open-argument-error.patch \
   patches/gtk+ patches/gtk+/01-mousewheel.patch \
   patches/zlib patches/zlib/01-shared-lib-support.patch \
   patches/zlib/02-cross-build.patch \
